### DISCUSSION TITLE
Fetch metadata from Google Workspace IdP?

### DISCUSSION THREAD

**Discussion by vtomasr5** [zitadel/zitadel#5711] [General]
How can I fetch custom attributes (aka metadata) from Google Workspace IdP?

I set custom attributes to the google user and I'm trying to fetch that metadata from Zitadel via Identity Provider and add claim mappings to use that attributes from Google. 

I tried to add metadata to a zitadel user and it worked as described [here](https://zitadel.com/docs/guides/manage/customize/user-metadata). But my goal is to get it from Google IdP and use Zitadel as a forward IdP for now.

Is that posible? If so, how can I do the claim mapping? Do I have to do it via Actions? If so, how? Do you have any example for that use case?

Thanks!

**Comment by hifabienne** [comment]
Hei @livio-a 
Can you help with the question?

**Comment by vterdunov** [comment]
Same question 🙏
